ash/Cargo.toml

```diff
@@ -15,11 +15,13 @@ rust-version = "1.59.0"
 libloading = { version = "0.7", optional = true }
 
 [features]
-default = ["loaded", "debug"]
+default = ["loaded", "debug", "std"]
+# Whether ash depends on the standard library. Disabling this raises the MSRV to 1.64.
+std = []
 # Link the Vulkan loader at compile time.
 linked = []
 # Support searching for the Vulkan loader manually at runtime.
-loaded = ["libloading"]
+loaded = ["std", "libloading"]
```

Comment:
Libloading and/or our logic requires `std` because of the `Arc`?

Reply:
libloading requires `std`, `Arc` is actually in `alloc` assuming your system has the requisite atomics support